extensions/dacpac/src/wizard/dataTierApplicationWizard.ts

```diff
@@ -310,11 +294,7 @@ export class DataTierApplicationWizard {
 			description: ''
 		};
 
-		const result = await service.generateDeployScript(this.model.filePath, this.model.database, ownerUri, azdata.TaskExecutionMode.script);
-		if (!result || !result.success) {
-			vscode.window.showErrorMessage(
-				localize('alertData.deployErrorMessage', "Deploy failed '{0}'", result.errorMessage ? result.errorMessage : 'Unknown'));
-		}
+		await service.generateDeployScript(this.model.filePath, this.model.database, ownerUri, azdata.TaskExecutionMode.script);
```

Comment:
Just to understand.. we don't care about showing message as per result anymore because this returns immediately after creating the thread in SqlToolsService? what if it fails in starting the second thread itself - will it not have that info?

Reply:
Yup. If there's an error message from before the thread is created in SqlToolsService, the error shows in the console. The previous error message showing code also wouldn't show these errors since the returned result would be undefined.